client/reader/sidebar/index.jsx

```diff
@@ -254,22 +235,51 @@ const ReaderSidebar = React.createClass( {
 					onTagExists={ this.highlightNewTag }
 					currentTag={ this.state.currentTag } />
 
-				</SidebarRegion>
+			</SidebarRegion>
 
-				{ this.renderAppPromo() }
+			{ this.props.shouldRenderAppPromo &&
```

Comment:
We noticed that, I was reluctant to do it in this PR as it would flag way more lines changed then actually were. I can do it as a separate PR though.